misc/docker/Dockerfile

```diff
@@ -1,23 +1,22 @@
 FROM busybox
 
 RUN set -x \
-    && wget -O- http://nixos.org/releases/nix/nix-1.9/nix-1.9-x86_64-linux.tar.bz2 | \
+    && wget -O- http://nixos.org/releases/nix/nix-1.10/nix-1.10-x86_64-linux.tar.bz2 | \
         bzcat - | tar xf - \
     && echo "nixbld:x:30000:nixbld1,nixbld10,nixbld2,nixbld3,nixbld4,nixbld5,nixbld6,nixbld7,nixbld8,nixbld9" >> /etc/group \
     && for i in $(seq 1 9); do echo "nixbld$i:x:3000$i:30000:::" >> /etc/passwd; done \
```

Comment:
doesn't this need to be `$(seq 1 10)` given that `nixbld10` exists in `/etc/group`